includes/blast_ui.form_per_program.inc

```diff
@@ -15,6 +15,14 @@
  */
 function blast_ui_per_blast_program_form($form, $form_state) {
 
+  // Add a warning, if need be (to be used for temporary messages like down-for-maintanence)
```

Comment:
This should mimic a typical drupal message to ensure consistent themeing.

```php
$form['warning'] = array(
  '#markup' => t(variable_get('blast_ui_warning_text', ''),
  '#prefix' => '<div class="messages warning">',
  '#suffix' => '</div>',
);
```

Reply:
The motivation for me is consistency which I am a bit obsessed about. All messages to the user should be the same then if we don't like them we can change them once instead of repeatedly ;-)